schema/article.js

```diff
@@ -18,9 +18,18 @@ const ArticleType = new GraphQLObjectType({
     title: {
       type: GraphQLString,
     },
+    published_at: {
+      type: GraphQLString,
+    },
+    updated_at: {
+      type: GraphQLString,
+    },
```

Comment:
The date field is not for any schema in particular, it just exposes a moment interface so that you can format the dates inside of the query.